Makefile

```diff
@@ -23,7 +23,12 @@ coverage: ## Generate code coverage
 	$(GOTOOL) cover -html=c.out -o $(OUTPUT_DIR)$(PACKAGE)-coverage.html
 
 lint:
-	$(GOLINT) -set_exit_status $(GOPKGS)
+# Implementing methods from the AWS SDK containing "Url" in their name causes lint warnings.
+	@{ \
+		results=$$($(GOLINT) -min_confidence=0.5 $(GOPKGS) 2>&1); \
+		filtered=$$(echo "$$results" | grep -v 'method GetQueueUrl'); \
+		echo $$filtered; \
+	}; [ -z "$$filtered" ]
```

Comment:
@sameersbn I had to resort to this hack because `golint` doesn't support `// nolint` comments in code.
Maybe we can get rid of this if we ever adopt the [golangci](https://github.com/golangci/golangci-lint) tooling.

Reply:
I am okay with migrating to `golangci` with this change. I'll submit a changeset against this PR.

Reply:
I've created https://github.com/antoineco/triggermesh-aws-event-sources/pull/2 where I have added `golangci-lint` to the ci env and reverted this change. There are some lint errors after this change which I'll let you resolve.

Reply:
note that i've added `make lint` to the CI testing.

Reply:
there was a typo in the ci config which was not caught by `circleci config validate`. https://github.com/antoineco/triggermesh-aws-event-sources/pull/3 should fix execution of the test job